spec/controllers/welcome_controller_spec.rb

```diff
@@ -36,12 +35,50 @@ def mock_user_sign_in
     allow(controller).to receive(:current_admin).and_return(admin)
   end
 
+  describe 'GET #index' do
+  before do
+    mock_user_sign_in
+    get :index
+  end
+
+  it 'renders the index template' do
+    expect(response).to render_template(:index)
+  end
+
+  context 'when no files are present' do
+    before do
+      test_upload_path = "#{Rails.root}/tmp/test_uploads"
+      FileUtils.mkdir_p(test_upload_path) 
```

Comment:
Git is saying there's some trailing whitespace and stuff.
Try running rubocop once maybe.
Also the credentials file line ending might have changed. Can you just do 
`git config --global core.autocrlf input`
This should ensure line ending are in linux format. Otherwise when we're all commiting, it will keep changing back and forth since we're using different OS's